### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by PauloMorillo**
We did get and count methods with their test to check our code and we try to do good comments for our modules

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
 """
-Contains the class DBStorage
+Contains the class DBStorage to save in database
```
Comment (JavierGutierrezC) [inline models/engine/db_storage.py:3]: What type of objects are returned in the objects list? and please check the comments in the count function

**Review by JavierGutierrezC — COMMENTED** [review]
What type of objects are returned in the objects list? and please check the comments in the count function

File: models/engine/db_storage.py (hunk 1 above)
Comment (PauloMorillo) [reply models/engine/db_storage.py:3]: we are returning all objects for each class (User, Amenity, City, Place, State, Review) and we are going to do better our comments for next one

**Review by JavierGutierrezC — APPROVED** [review]
What type of objects are returned in the objects list? and please check the comments in the count function

**Pull request merged**